contracts/libs/RatioUtils.sol

```diff
@@ -52,10 +52,10 @@ library RatioUtils {
     }
 
     function radToWad(uint256 _rad) internal pure returns (uint256 _wad) {
-        _wad = (_rad * WAD) / RAD;
+        _wad = _rad.mul(WAD).div(RAD);
```

Comment:
let it be, but actually not needed in compiler above 0.8 * / + - are safe

Reply:
they are not safe, just prevent overflows & underflows